tests/js/pipeline-operator/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -391,5 +435,29 @@ const searchResults$ =
 
 v |> #.method() |> f;
 
+async function* f() {
+  return x |> (yield #) |> (yield y) |> (yield) |> yield;
+}
+
+async function* f() {
+  return x |> await # |> (await y);
+}
+
+async function* f() {
+  return x |> (y) |> y;
```

Comment:
In which one of the [three modes](https://babeljs.io/docs/en/babel-plugin-proposal-pipeline-operator#with-a-configuration-file-recommended) are these parens significant?

Reply:
Looks like we should rethrow this error instead of preserving parens: `Pipeline is in topic style but does not use topic reference`